docs/configurations/distribute.md

```diff
@@ -1,9 +1,4 @@
-# Long polling
-*experimental*
```

Comment:
you should add distribute.md, do not modify long polling file and add a new long polling file